src/layouts/Tabbed/helpers.js

```diff
@@ -0,0 +1,6 @@
+import { config } from 'storybook-addon-designs'
+
+export const figmaConfig = (nodeId) => config({
+  type: 'figma',
+  url: `https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=${nodeId}`
+})
```

Comment:
If some code solves a separate problem, it should be a separate PR. Makes reviewing easy for us and counts as more merged PRs for you.